control-plane/connect-inject/peering_dialer_controller.go

```diff
@@ -43,16 +44,32 @@ func (r *PeeringDialerController) Reconcile(ctx context.Context, req ctrl.Reques
 	// If the PeeringDialer resource has been deleted (and we get an IsNotFound
 	// error), we need to delete it in Consul.
 	if k8serrors.IsNotFound(err) {
-		r.Log.Info("PeeringDialer was deleted, deleting from Consul", "name", req.Name, "ns", req.Namespace)
-		if err := r.deletePeering(ctx, req.Name); err != nil {
-			return ctrl.Result{}, err
-		}
+		r.Log.Info("PeeringDialer resource not found. Ignoring.", "name", req.Name, "ns", req.Namespace)
```

Comment:
```suggestion
		r.Log.Info("PeeringDialer resource not found. Ignoring resource", "name", req.Name, "ns", req.Namespace)
```
super nit: just cause we don't have a . at the end of any other logs :)